CONTRIBUTING.md

```diff
@@ -107,10 +107,59 @@ approved, other developers with commit access may commit it for the author. When
 doing so, it is important to retain correct attribution of the contribution.
 Generally speaking, Git handles attribution automatically.
 
+## Development
+
+### Local setup
+
+Install [pyenv](https://github.com/pyenv/pyenv#installation) and [poetry](https://python-poetry.org/docs/#installation)
+
+Install and set python 3.7.8 with pyenv:
```

Comment:
can also use a newer version though

Reply:
As long as the contributor is using one of the "supported" Python versions (one of the ones tested in your CI), does it matter which specific one they use? What if they want to use 3.7.7? Or 3.7.12? For that matter, what if they don't want to use pyenv? It should be totally legitimate to use packages straight from python.org, or from Homebrew, or from the `asdf` package manager for example.

The goal here should be to reduce friction for potential contributors, and I could see dictating specific tools and versions like this as increasing friction, making it more difficult for a new contributor to get started, meaning they may decide it's not worth it.

Reply:
updated the doc